drivers/mge-hid.c

```diff
@@ -1438,6 +1456,9 @@ static hid_info_t mge_hid2nut[] =
 	{ "input.bypass.frequency", 0, 0, "UPS.PowerConverter.Input.[2].Frequency", NULL, "%.1f", 0, NULL },
 	{ "input.bypass.frequency.nominal", 0, 0, "UPS.Flow.[2].ConfigFrequency", NULL, "%.0f", HU_FLAG_STATIC, NULL },
 
+	/* ECO(HE) Mode switch , maybe better = outlet.5.switchable ? */
+	{ "input.eco.switchable", ST_FLAG_RW | ST_FLAG_STRING, 8, "UPS.PowerConverter.Input.[5].Switchable", NULL, "%.0f", HU_FLAG_SEMI_STATIC, NULL },
```

Comment:
@arnaudquette-eaton is it right definition , what else needed?

Reply:
ESS Maybe Converter Mode ?

Reply:
some info on ESS: https://www.eaton.com/content/dam/eaton/products/backup-power-ups-surge-it-power-distribution/backup-power-ups/portfolio/eaton-ess-technology-brief-brochure.pdf

Reply:
Thanks a lot, do you think i can compile and test now if UPS will switch to ECO by  `upsrw` and `input.eco.switchable` to set mode 1 ?

Reply:
up to you, you're the dev and the tester ;)